src/qibolab/instruments/zhinst/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .executor import Zurich
+from .pulse import ZhPulse
+from .sweep import ProcessedSweeps, classify_sweepers
+from .util import acquire_channel_name, measure_channel_name
```

Comment:
This is perfectly fine in general, but to move towards #790 I'd suggest you to define a local index (`__all__` variable) in each file, and limit `__init__.py` to:
```suggestion
from .executor import *
from .pulse import *
from .sweep import *
from .util import *
```

Reply:
If needed, we could also use this scheme to automatically generate `__all__` for subpackages. Cf.
https://github.com/numpy/numpy/blob/a8d4e48498a725306ff7e0c0460ab23428feb510/numpy/fft/__init__.py#L210-L211

However, if there is nothing else in the subpackage than what you want to export, there is no need for an explicit `__all__`.